src/components/data/FileBrowser.js

```diff
@@ -248,6 +249,13 @@ const BucketBrowserTable = ({
                             },
                             [label]
                           ),
+                          h(ClipboardButton, {
+                            tooltip: 'Copy file URL to clipboard',
+                            className: 'cell-hover-only',
+                            style: { marginLeft: '1ch' },
+                            text: `gs://${bucketName}/${object.name}`,
+                            iconSize: 14,
```

Comment:
This PR has a beautiful writeup as to _why_ the `iconSize` is 14, maybe it'd be worth memorializing it as a comment with a link back to this PR, so someone like me doesn't blindly bumble in here and change or remove the value?

Reply:
Good idea! Just added a comment inline linking back here.